src/components/ActionModal/ActionModal.js

```diff
@@ -28,28 +28,36 @@ import Icon from 'components/Icon';
 import { spacing, fontStyles } from 'utils/variables';
 import type { Theme } from 'models/Theme';
 
+
```

Comment:
What's the rule for these extra spaces?

Reply:
I've been following this: https://pillarwallet.slack.com/archives/CEEV48867/p1583529140199300

Reply:
Nice! Totally missed that one. Thanks!